spec/providers/database.spec.ts

```diff
@@ -457,8 +511,15 @@ describe('Database Functions', () => {
     const apps = new appsNamespace.Apps();
 
     const populate = (data: any) => {
-      const [instance, path] = database.resourceToInstanceAndPath(
-        'projects/_/instances/other-subdomain/refs/foo'
+      const [instance, path] = database.extractInstanceAndPath(
+        {
+          context: {
+            resource: {
+              name: 'projects/_/instances/other-subdomain/refs/foo'
+            },
+            domain: 'firebaseio-staging.com'
```

Comment:
I was trying with the multi-region path. However, it didn't work due to a bug in the admin SDK.
I will fix that separately. For now, this will unblock testing in staging at least.